src/conf.c

```diff
@@ -581,25 +581,6 @@ int ConfValIsFalse(const char *val)
  * \retval 1 will be returned if the name is found and was properly
  * converted to a double, otherwise 0 will be returned.
  */
-int ConfGetDouble(const char *name, double *val)
```

Comment:
These non-static `Conf` functions are somewhat public API, as plugin users can make use of them. This one is likely not used, but maybe?

Reply:
Are you sure it works ?
It looks like my compiler optimized these unused public API functions and that plugins cannot find them at runtime...